version.gradle

```diff
@@ -1,3 +1,3 @@
 ext {
-  PROSPECTIVE_RELEASE_VERSION = '0.8.3'
+  PROSPECTIVE_RELEASE_VERSION = '0.9.0'
```

Comment:
:open_mouth:

Reply:
As per convention... minor is bumped when the IDE compatibility range is extended